lib/rest2.js

```diff
@@ -886,6 +886,24 @@ class RESTv2 {
     })
   }
 
+  /**
+   * Changes the collateral value of an active derivatives position for a certain pair.
+   *
+   * @param {string} symbol
+   * @param {number} collateral
+   * @param {Method?} cb - legacy callback
+   * @return {Promise} p
+   * @see https://docs.bitfinex.com/v2/reference#rest-auth-deriv-pos-collateral-set
+   */
+
+  derivsPositionCollateralSet (symbol = 'tBTCF0:USDF0', collateral, cb) {
+    const url = '/auth/w/deriv/collateral/set'
+    const isTrue = (res) => !!(res && res[0] && res[0][0])
```

Comment:
I might be being a little pedantic here but could we change the name of the this to something like `isRequestValid`?

Reply:
sure!